iis/iis.go

```diff
@@ -145,6 +175,46 @@ func getVersion() (string, error) {
 	}
 }
 
+// Gets a version object of InetMgr.exe which parses major.minor.build.revision string
+func getVersion() (*iisVersion, error) {
+	versionStr, err := getVersionStr()
+
+	if err != nil {
+		return nil, fmt.Errorf("Failed to get version string for iisVersion parsing: %v", err)
+	}
+
+	versionNumbers := strings.Split(versionStr, ".")
+	if len(versionNumbers) != 4 {
+		return nil, fmt.Errorf("Format of IIS version is improper. It must have \"major.minor.build.revision\" format")
+	}
+	version := &iisVersion{}
+	if major, err := strconv.Atoi(versionNumbers[0]); err != nil {
+		return nil, fmt.Errorf("Failed to set Major version number: %v", err)
+	} else {
+		version.Major = major
+	}
```

Comment:
Just a suggestion, feel free to ignore it.

```
major, err := strconv.Atoi(versionNumbers[0])
if err != nil {
 		return nil, fmt.Errorf("Failed to set Major version number: %v", err)
} 
 version.Major = major
 ```

We can remove `else` from all 4 conditions {`major`, `minor`, `build` and `revision`}